### ISSUE TITLE
很棒的项目，为什么添加了Tianji Reporter ，在服务器状态里看不到机器？

### ISSUE THREAD

**Issue by neilzhangpro** [msgbyte/tianji#29]
`[Info] Write to systemd configuration
Created symlink /etc/systemd/system/multi-user.target.wants/tianji-reporter.service → /lib/systemd/system/tianji-reporter.service.
[Info] Tianji Reporter Started!`
很棒的项目！
想了解为什么服务器上显示已经安装好了，在工作台上依旧看到不服务器状态呢？

**Comment by moonrailgun** [comment]
maybe you should check out tianji reporter service log, the reason is too much.

For example:
```bash
journalctl -fu tianji-reporter.service
```

**Comment by neilzhangpro** [comment]
thx adn yep,i can see somthing like:
`Feb 26 01:07:30 C20230707097180 tianji-reporter[604032]: 2024/02/26 01:07:30 Send report data to: https://xxxxxxx:8001
Feb 26 01:07:35 C20230707097180 tianji-reporter[604032]: 2024/02/26 01:07:35 [Report] {"workspaceId":"clnzoxcy10001vy2ohi4obbi0","name":"C20230707097180","hostname":"C20230707097180","timeout":10,"payload":{"uptime":313663,"load":1.19,"memory_total":2032708,"memory_used":1240844,"swap_total":1049596,"swap_used":500328,"hdd_total":40148,"hdd_used":21116,"cpu":74.6,"network_tx":31948,"network_rx":1047482,"network_in":4752749176,"network_out":1206818013}}`
in my server console,its works. but i cant find it at my Web Dashboard which in http://xxxx.com/servers,what should i do ?

**Comment by neilzhangpro** [comment]
ps: some websoket error show `index-p9xOC6fQ.js:509 WebSocket connection to 'wss://xxxx.com:8001/socket.io/?EIO=4&transport=websocket' failed: `

**Comment by neilzhangpro** [comment]
hi i have fix it , you must add a roule in your nginx like this:
`location /socket.io/ {
        proxy_pass http://xxxx.com:12345/socket.io/;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "Upgrade";
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_read_timeout 86400;  # this setting is necessary for WebSocket connections to work
    }`
i think its better add in your docker config~ thx anyway!

**Comment by moonrailgun** [comment]
yeah, gateway need add websocket support. i will add it in document. thanks for your report
